packages/api/functions/express/routes/program.js

```diff
@@ -20,20 +20,24 @@ programRouter.post('/', wrapAsync(async (req, res) => {
 
 programRouter.put('/:programId', wrapAsync(async (req, res) => {
   const { programId } = req.params
+  const userId = req.cognitoUser.id
```

Comment:
what happens if thhe API is called unauthenticated?

Reply:
API Gateway Cognito Authorizer prevents it from reaching Lambda